Content.Client/UserInterface/GameHud.cs

```diff
@@ -267,6 +267,8 @@ private void ButtonTutorialOnOnToggled()
             {
                 _tutorialWindow.OpenCentered();
                 _buttonTutorial.Pressed = true;
+                //Fixes scrollbar not showing up until window is resized/scrolled.
+                _tutorialWindow.ForceRunLayoutUpdate();
```

Comment:
This really should not be the correct way to fix this issue. The documentation for `ForceRunLayoutUpdate` specifically says that needing to use it most certainly signifies a bug in the UI system, and this is one such place I feel that holds.

Reply:
Will see if I can find the issue and make a separate PR with a fix.